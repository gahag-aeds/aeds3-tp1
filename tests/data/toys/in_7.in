8
0.0000 0.92397 0.44736 0.72287 0.43258 0.39259 0.3255 0.03555 
0.07603 0.0000 0.02781 0.95599 0.00963 0.30469 0.39248 0.46535 
0.55264 0.97219 0.0000 0.28504 0.60933 0.14872 0.17444 0.23458 
0.27713 0.04401 0.71496 0.0000 0.71037 0.34229 0.48981 0.66157 
0.56742 0.99037 0.39067 0.28963 0.0000 0.69527 0.24041 0.62552 
0.60741 0.69531 0.85128 0.65771 0.30473 0.0000 0.73374 0.3525 
0.6745 0.60752 0.82556 0.51019 0.75959 0.26626 0.0000 0.50318 
0.96445 0.53465 0.76542 0.33843 0.37448 0.6475 0.49682 0.0000 
