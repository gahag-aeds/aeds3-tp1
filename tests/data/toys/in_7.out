0.081728 0.045060 0.060935 0.115289 0.143915 0.187561 0.179586 0.185924 